package.json

```
@@ -0,0 +1,77 @@
+{
+  "name": "cosmiconfig-typescript-swc-loader",
+  "version": "0.0.1",
+  "description": "Fast TypeScript config file handler for cosmiconfig using SWC",
+  "main": "dist/cjs/index.js",
+  "repository": "git@github.com:charlypoly/cosmiconfig-typescript-swc-loader.git",
+  "author": "Charly POLY <cpoly55@gmail.com>",
+  "license": "MIT",
+  "scripts": {
+    "test": "jest --passWithNoTests",
+    "clean": "rimraf dist",
+    "prebuild": "yarn clean",
+    "build": "bob build --single",
+    "prettier": "prettier --write --list-different .",
+    "prettier:check": "prettier --check .",
+    "lint": "eslint .",
```

```suggestion
    "lint": "eslint --ignore-path .gitignore .",
```
and `.eslintignore` can be removed